public/content/documentation/how-to-test/components/not-sure-if-it-is-an-issue.md

```diff
@@ -0,0 +1,217 @@
+## General Notes
+
+When conducting accessibility testing, testers may come across barriers that could be accessibility-related issues, or they could be unrelated. If you're unsure whether the problem you've identified is a genuine accessibility issue, the following steps can help you verify it.
+
+## Developer Notes
+
+Accessibility testing is important because it helps to ensure that everyone can use a website or application, regardless of their disability. This includes people who are blind, deaf, have low vision, or have mobility impairments.
+
+To thoroughly test web accessibility, a combination of manual and automated testing methods are performed against the [Web Content Accessibility Guidelines (WCAG) 2.2 Level A and AA](https://www.w3.org/TR/WCAG22/). Manual testing involves people navigating the site, using assistive technologies like screen readers or keyboard-only navigation, and verifying if all content is perceivable, operable, understandable. Automated testing utilizes specialized tools to scan for common accessibility issues, such as missing alternative text for images or improper heading structure.
+
+By combining both approaches, we can identify and address accessibility barriers comprehensively, improving the overall user experience and inclusivity of our digital products.
+
+## 1. **Test against MagentaA11y.com**
```

Comment:
As discussed on our call, reconfigure this list items something like this: 

1. ## Test against MagentaA11y.com

And then indent the heck out of the content within that list item.